src/test/java/com/notably/commons/core/Path/AbsolutePathTest.java

```diff
@@ -0,0 +1,38 @@
+package com.notably.commons.core.path;
+
+import static com.notably.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import com.notably.commons.core.path.exceptions.InvalidPathException;
+
+class AbsolutePathTest {
+
+    @Test
+    public void createAbsolutePath() {
+        assertThrows(InvalidPathException.class, () -> AbsolutePath.fromString("CS2103/notes"));
+        try {
+            assertEquals(new AbsolutePath("/CS2103/notes"), AbsolutePath.fromString("/CS2103/notes"));
+        } catch (InvalidPathException ex) {
+            System.out.println("Error");
+        }
+
+    }
+
+
+    @Test
+    public void convertAbsolutePath() {
+        AbsolutePath absolutePath = new AbsolutePath("/CS2103/notes/hello");
+        AbsolutePath currPath = new AbsolutePath("/CS2103");
+        RelativePath relativePath = new RelativePath("notes/hello");
+        try {
+            AbsolutePath expectedAbsolutePath = new AbsolutePath("/CS2103/notes/hello");
+            assertEquals(expectedAbsolutePath, AbsolutePath.fromRelativePath(relativePath, currPath));
+            assertEquals(new RelativePath("notes/hello"), absolutePath.toRelativePath(currPath));
+        } catch (InvalidPathException ex) {
+            System.out.println("Error");
+        }
+
```

Comment:
Just FYI!

I believe usually test cases are organized this way (can't remember where I read about it, so please take this advice with a grain of salt LOL):
1. Setup all the parameters used in the assertions
2. Setup all the expected variables
3. Do the assertions

You can take a look at my PR #110, I basically organized it that way!